authentication/handler.go

```diff
@@ -0,0 +1,205 @@
+package authentication
+
+import (
+	"context"
+	"errors"
+	"net/http"
+	"strings"
+	"sync"
+
+	"github.com/SKF/go-rest-utility/problems"
+	"github.com/SKF/go-utility/v2/accesstokensubcontext"
+	"github.com/SKF/go-utility/v2/jwk"
+	"github.com/SKF/go-utility/v2/jwt"
+	"github.com/SKF/go-utility/v2/log"
+	"github.com/SKF/go-utility/v2/stages"
+	"github.com/SKF/go-utility/v2/useridcontext"
+	"github.com/gorilla/mux"
+	old_errors "github.com/pkg/errors"
+	"go.opencensus.io/trace"
```

Comment:
I guess it should import the common trace pkg instead of `go.opencensus.io/trace`